pkg/build/buildutil/buildutil_test.go

```diff
@@ -1 +1,64 @@
 package buildutil
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+func TestNameForBuildVolume(t *testing.T) {
+	type args struct {
+		objName string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Secret One",
+			args: args{objName: "secret-one"},
+			want: "secret-one-userBuildVolume",
+		},
+		{
+			name: "ConfigMap One",
+			args: args{objName: "configmap-one"},
+			want: "configmap-one-userBuildVolume",
+		},
```

Comment:
Need a unit test if the user-provided build volume name is > 47 characters. Note that the namer has a hash function, you'll need to figure out what the input's hash is to get the test passing.

```suggestion
		},
		{
			name: "Greater than 47 characters",
			args: args{objName: "build-volume-larger-than-47-characters-but-less-than-63"},
			want: "build-volume-larger-than-47-character-XXXXXXXX-userBuildVolume",
		},
```